jelixir/lib/jelixir.ex

```diff
@@ -20,6 +24,78 @@ defmodule JelixirLib do
     end
   end
 
+  def conver(task, file_name_string) when task == :schema do
+    with {:read_file_result, {:ok, file_content}} <-
+           {:read_file_result, read_file(file_name_string, ".ex")} do
+      if !File.exists?(@default_folder_out) do
+        File.mkdir(@default_folder_out)
+      end
+
+      parse_schema_result =
+        String.split(file_content, "\n")
+        |> Enum.map(fn line ->
+          line |> String.trim() |> read_line_schema()
+        end)
+        |> Enum.filter(fn item -> item != {:skip, nil} end)
+        |> Enum.into(%{})
+
+      module_name = parse_schema_result["module_name"]
+      schema_name = parse_schema_result["schema_name"]
+
+      node_result =
+        parse_schema_result
+        |> Map.delete("module_name")
+        |> Map.delete("schema_name")
```

Comment:
I think it should be `node_result = Map.drop(parse_schema_result, ["module_name", "schema_name])`

Reply:
@quangvo09 Thank you for your suggestion.